lib/feedsme/index.js

```diff
@@ -316,12 +362,15 @@ Feedsme.prototype._triggerOneDep = async function _triggerOneDep({ publish, prom
  * @returns {Object} strategy and whether we should trigger
  * @api private
  */
-Feedsme.prototype._triggerStrategy = function _triggerStrategy({ releaseLine, rootPkg, pkg, env }) {
+Feedsme.prototype._triggerStrategy = function _triggerStrategy({ registryRootPkg, releaseLine, rootPkg, pkg, env }) {
   if (env !== 'dev' && releaseLine) return { strategy: 'release', trigger: true };
   if (!releaseLine) return { strategy: 'legacy', trigger: true };
 
   const { name, version: rootVersion } = rootPkg;
   const { dependencies } = pkg;
+  const rootPkgVersions = Object.keys(registryRootPkg.versions || {})
+    .sort((y, x) => semver.compare(x, y));
```

Comment:
Yes, since it is sorting in descending order – but a comment would be helpful. Also I'd suggest inverting the invert (see my other comment below) 

To prove the descending order point ... e.g.

``` js
const versions = [ '0.0.0',
  '0.1.0',
  '0.2.0',
  '0.3.0',
  '0.3.1',
  '0.3.2',
  '0.3.3',
  '0.4.0',
  '0.5.0',
  '0.6.0',
  '0.7.0',
  '0.8.0',
  '0.8.1',
  '0.8.2',
  '1.0.0',
  '1.1.0',
  '1.1.1',
  '1.1.2',
  '1.1.3',
  '1.1.4',
  '1.1.5',
  '1.2.0',
  '1.2.1',
  '1.2.2',
  '1.2.3',
  '1.3.0',
  '1.4.0',
  '2.0.0' ];

console.log(versions.sort((y, x) => semver.compare(x, y));
```

which outputs:

```
[ '2.0.0',
  '1.4.0',
  '1.3.0',
  '1.2.3',
  '1.2.2',
  '1.2.1',
  '1.2.0',
  '1.1.5',
  '1.1.4',
  '1.1.3',
  '1.1.2',
  '1.1.1',
  '1.1.0',
  '1.0.0',
  '0.8.2',
  '0.8.1',
  '0.8.0',
  '0.7.0',
  '0.6.0',
  '0.5.0',
  '0.4.0',
  '0.3.3',
  '0.3.2',
  '0.3.1',
  '0.3.0',
  '0.2.0',
  '0.1.0',
  '0.0.0' ]
```

Reply:
@indexzero @msluther i guess we could use `rcompare` to make it more clear.